docs/source/release_notes/release-notes.rst

```diff
@@ -69,6 +69,11 @@ develop
          - Cassandra truncates that are going to fail will do so faster.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1660>`__)
 
+    *    - |devbreak|
+         - The backup lock endpoints now use ``PersistentLockId`` instead of ``LockEntry``. Additionally, the
+           ``releaseLock`` endpoint will now throw the http-remoting compatible exceptions on errors and not return anything on success.
+           (`Pull Request <https://github.com/palantir/atlasdb/pull/1665>`__)
```

Comment:
For part two, we should outline what the previous behaviour was, and also what dev action (if any) is required as a result of that. 

I think I'd prefer "returns an exception object compatible with http-remoting clients", with a link to http-remoting (especially because we don't normally use it in AtlasDB) as opposed to "throw the http-remoting compatible exceptions" too.